elastic/kibana
# [Inventory] Add Sharing button

## Summary

Adds a Share link button to the Inventory page, next to the Add Data button on the top right of the screen. This creates a short link and copies it to the user's clipboard so they can share it.

https://github.com/user-attachments/assets/42c7b2f9-da19-4ced-8d28-211234bea4cf

## How to Test

- Log in and go to Inventory page. Ensure there is data loaded and entities are showing.
- Put a filter on the inventory to restrict the results to a given state, such as `entity.type : "service"`.
- Click on the `Share` button and wait until you get a success toast message.
- Open a new empty tab on the browser and paste the link into the url bar to navigate to the page.

**Expected Result**: The page should navigate to the Inventory page and load into the same state as the page on the original browser tab.

Closes #192325
